app/helpers/collection_helper.rb

```diff
@@ -1,4 +1,9 @@
 module CollectionHelper
+  # strip leading 'a' from collection catkeys
+  def self.strip_leading_a(collection_id)
```

Comment:
Can you say a little bit about using a class method here vs. an instance method? It looks like most of the current helper methods are instance methods.

Reply:
From what I can tell you can't use an instance method in concerns without importing the entire helper method. So I went with a class method so that we could just call the class method in the concern files.

Reply:
Gotcha, thank you!